posthog/models/project.py

```diff
@@ -60,6 +60,9 @@ def __str__(self):
 
     __repr__ = sane_repr("id", "name")
 
-    @cached_property
-    def passthrough_team(self) -> "Team":
-        return self.teams.get(pk=self.pk)
+    @lru_cache(maxsize=1)
+    def get_passthrough_team(self, visible_team_ids: tuple[int, ...]) -> "Team":
```

Comment:
So this is meant to be like the default team?

Reply:
Yup